docs/examples/viz_helical_motion.py

```diff
@@ -68,94 +71,130 @@
 directions = np.array([[1, 0, 0]])
 heights = np.array([8])
 arrow_actor = actor.arrow(centers, directions, color_arrow, heights,
-                          resolution=20, tip_length=0.06, tip_radius=0.012,
+                          resolution=50, tip_length=0.06, tip_radius=0.012,
                           shaft_radius=0.005)
+arrow_actor.GetProperty().SetAmbient(1)
 scene.add(arrow_actor)
 
 ###############################################################################
 # Initializing the initial coordinates of the particle
 
 x = initial_velocity*time + 0.5*acc*(time**2)
-y = np.sin(angular_frq*time + phase_angle)
-z = np.cos(angular_frq*time + phase_angle)
+y = radius_path*np.sin(angular_frq*time + phase_angle)
+z = radius_path*np.cos(angular_frq*time + phase_angle)
+pts = np.array([[x, y, z]])
+
+
+###############################################################################
+# Initializing the path_actor which will trace the path followed by the
+# charged particle
+
+num_total_steps = 200  # total number of simulation steps
+origin = np.asarray([0, 0, 0], dtype=float)
+position = np.tile(origin, (num_total_steps, 1))
+
+v = np.linspace(0, 1, num_total_steps)
+cmap_name = 'spring'  # using colormap to color the path of the particle
+colors = colormap.create_colormap(v, name=cmap_name)
+
+path_actor = actor.line([position], colors, linewidth=3)
+path_actor.position = position
+path_actor.vertices = utils.vertices_from_actor(path_actor)
+path_actor.vcolors = utils.colors_from_actor(path_actor)
+print(len(path_actor.vcolors))
+path_actor.no_vertices_per_point = \
+    len(path_actor.vertices) / num_total_steps
+path_actor.initial_vertices = path_actor.vertices.copy() - \
+    np.repeat(position, path_actor.no_vertices_per_point, axis=0)
```

Comment:
@SunTzunami please make a utility function called ``attributes_to_actor`` in ``fury.actor`` and move these operations (lines 101-105) there.

Reply:
The function should take as input an actor and return the actor with the new attributes as you did above.

Reply:
Created attributes_to_actor.